docs-ref-conceptual/azure-cli-sp-tutorial-7.md

```diff
@@ -16,17 +16,62 @@ keywords: azure service principal, create service principal azure, create servic
 
 If you lose your service principal credentials, reset the credentials using [az ad sp credential reset](/cli/azure/ad/sp/credential#az-ad-sp-credential-reset). This step details how to reset your service principal password or certificate.
 
+> [!WARNING]
+> While learning to manage Azure service principals, returning passwords and credential locations in your terminal, and consequently in your log file, is a solution. However, when outside of a testing environment, [store credential output in a variable](#reset-credentials-storing-output-in-a-variable).
+
+## Basic credential reset
+
+Reset a service principal credential with a password. If you don't know the ID associated with your service principal, use the `az ad sp list` command as explained in [Get an existing service principal](./azure-cli-sp-tutorial-4.md).
+
 ```azurecli-interactive
-# reset a service principal credential with a password
 az ad sp credential reset --id myServicePrincipalID
 ```
 
+Console output showing the new password in the console, and although not visible in the console, also in the log:
```

Comment:
This line is conflict.

Reply:
@bebound, please help me to understand. What should this sentence say instead?

Reply:
@bebound , I think I get it -- I'll reword this sentence to something like  "It is visible in the console and written to the log".